### PR TITLE
Spanish traduced

### PR DESCRIPTION
**Pull request by danicm47**

### CODE REVIEW

File: app/src/main/res/values-es-rES/strings.xml [hunk 1]
```diff
@@ -1,2 +1,90 @@
...
+    <string name="defaultName">John Doe</string>
+    <string name="defaultEmail">johndoe@gmail.com</string>
+    <string name="breed_title">Raza</string>
+    <string name="birth_date_title">Edad(años)</string>
+    <string name="gender_title">Género</string>
+    <string name="birth_title">Fecha de nacimiento</string>
+    <string name="weight_title">Peso(kg)</string>
+    <string name="patologies_title">Patologías</string>
+    <string name="recommended_kcal_title">Calorias recomendadas(Kcal)</string>
+    <string name="wash_title">Frecuencia de baño(days)</string>
```
Comment (enrichp) [inline app/src/main/res/values-es-rES/strings.xml]: (dias)

File: app/src/main/res/values-es-rES/strings.xml [hunk 2]
```diff
@@ -1,2 +1,90 @@
...
+    <string name="input_gender">Género</string>
+    <string name="input_birth_date">Fecha de nacimiento</string>
+    <string name="input_breed">Raza</string>
+    <string name="input_weight">Peso (kg)</string>
+    <string name="input_recommended_calories">Calorías recomendadas (kcal)</string>
+    <string name="input_wash_frequency">Frecuencia de baño (days)</string>
+    <string name="input_pathologies">Patologías</string>
+    <string name="select_birth_date">Selecciona fecha de nacimiento</string>
+    <string name="register_new_pet">Registrar nueva mascota</string>
+    <string name="error_empty_field">Todos los campos deben ser rellenados (patologías no incluída)</string>
```
Comment (AlbertPG) [inline app/src/main/res/values-es-rES/strings.xml]: You should put "(sin incluir patologías)".

**Review by AlbertPG — CHANGES_REQUESTED** [review]
You should change these strings below

**Review by AlbertPG — APPROVED** [review]
Okey, no problems

**Review by enrichp — APPROVED** [review]

**Pull request merged**